buildbot/master/files/config/factories.py

```diff
@@ -69,8 +70,11 @@ def __init__(self, builder_name, environment):
             print(str(e))
             dynamic_steps = [BadConfigurationStep(e)]
 
-        # TODO: windows compatibility (use a custom script for this?)
-        pkill_step = [steps.ShellCommand(command=["pkill", "-x", "servo"],
+        if is_windows():
+            pkill_command = ["powershell", "kill", "-n", "servo"]
+        else:
+            pkill_command = ["pkill", "-x", "servo"]
```

Comment:
Another option is a script in `etc/ci`, which neatly steps around the "how to detect Windows" problem.